app/views/shared/_news_signup_test.html.erb

```diff
@@ -1,4 +1,4 @@
-<div class="news-signup-test" data-dough-component="Newsletter">
+<div class="news-signup-test <% if local_assigns.has_key?(:visible) %>news-signup-test--with-image news-signup-test--in-article <% end %>" <% unless local_assigns.has_key?(:visible) %>data-dough-component="Newsletter"<% end %>>
```

Comment:
actually use a ternary and move the double quote outside of the conditional. how does that sound?